lib/ljglibs/cdefs/fontconfig.moon

```diff
@@ -0,0 +1,9 @@
+-- Copyright 2015 The Howl Developers
```

Comment:
This probably shouldn't be under ljglibs, as it's not part of the g* family as far as I can see. I think `lib/howl/cdefs` would be a better match.